tools/pom.xml

```diff
@@ -44,11 +44,6 @@
     </properties>
 
     <dependencies>
-        <dependency>
-            <groupId>com.nvidia</groupId>
-            <artifactId>rapids-4-spark-common_${scala.binary.version}</artifactId>
```

Comment:
Once this is removed, there's no need for a separate spark-common module.  That code can be folded into the plugin as it would be the only consumer.

Reply:
Filed #7177 to track removal of common module